aiogram/dispatcher/handler.py

```diff
@@ -112,7 +116,7 @@ def unregister(self, handler):
                     ctx_token = current_handler.set(handler_obj.handler)
                     try:
                         if self.middleware_key:
-                            await self.dispatcher.middleware.trigger(f"process_{self.middleware_key}", args + (data,))
+                            await self.dispatcher.middleware.trigger(self.formatted_middleware_key, args + (data,))
```

Comment:
`self.formatted_middleware_key` can't be used here because `process_<key>` is expected instead of `pre_process_<key>`